celer/tests/conv_warning/visualize_logs.py

```diff
@@ -0,0 +1,62 @@
+import re
```

Comment:
this is overcomplicated (+ if the error comes from a wrong gap/primal computation you won't see it), run the solver several times while increasing the maximum number of iterations instead

see in the 2 last cells here for example : https://github.com/gpeyre/numerical-tours/blob/master/python/optim_7_noncvx_pro.ipynb